package.json

```diff
@@ -80,7 +80,6 @@
     "config": "3.0.1",
     "connect-session-sequelize": "6.0.0",
     "connect-slashes": "^1.4.0",
-    "cookie-parser": "1.4.4",
```

Comment:
This is no longer required for express-session, in fact they recommend not using it if you don't need it for anything else (which we don't, eg. we don't use `req.cookies` anywhere).

Reply:
_Interesting_ 👍 always up  for removing dependencies